src/core/libmaven/databases.cpp

```diff
@@ -10,6 +10,9 @@ int Databases::loadCompoundCSVFile(string filename) {
     map<string, int> header;
     vector<string> headers;
 
+    // recent the contents of the vector containing the names of invalid rows
```

Comment:
@saifulbkhan "recent". Typo?

Reply:
Ehem. I think I meant to write "reset" maybe. Will correct.